package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "koa-npm-resolution",
```

Comment:
What do you think about the name `koa-node-resolve`?

Using the Node name feels a little more correct than NPM, since this style of module resolution is more principally a Node thing than an NPM thing.

Also, it would nicely align the name with https://www.npmjs.com/package/rollup-plugin-node-resolve, which is the same thing but for Rollup.

Reply:
Renamed package to `koa-node-resolve` - Will change the repo name now.